addons/survey/tests/test_survey.py

```diff
@@ -125,6 +127,91 @@ def test_answer_display_name(self):
                     question.suggested_answer_ids[0].id, **{'answer_value_row': question.matrix_row_ids[1].id})
                 self.assertEqual(question_answer_2.display_name, 'Column0: Row1')
 
+    def test_survey_session_speed_reward_config_propagation(self):
+        """Test the speed rating time limit propagation to non time-customized questions."""
+        test_survey = self.env['survey.survey'].create({
+            'title': 'Test This Survey',
+            'scoring_type': 'scoring_with_answers',
+            'question_and_page_ids': [
+                Command.create({
```

Comment:
Command is deprecated
```suggestion
                (0, 0, {
```